### DISCUSSION TITLE
STUNNED

### DISCUSSION THREAD

**Discussion by jessekrubin** [astral-sh/rye#1106] [General]
Hello rye team (@mitsuhiko et al)

I just started using rye and have been STUNNED at how well it works! It even works well on windows -- windows doesn't even work well on windows.

Really and truly excellent.

**Comment by kwongtn** [comment]
Don't think this should be an issue? Should go into discussions instead 🤔

**Comment by zanieb** [comment]
Glad you're enjoying it!
